frontend/views/components/CopyToClipboard.vue

```diff
@@ -0,0 +1,108 @@
+<template lang='pug'>
+.c-wrapper(@click='copyToClipboard')
+  input.c-invisible-input(
+    type='text'
+    ref='input'
+    :value='textToCopy'
+  )
+  slot
+  transition(name='fade')
+    .c-tooltip(v-if='ephemeral.tooltipActive') {{ L(tooltipText) }}
+</template>
+
+<script>
+import Tooltip from '@components/Tooltip.vue'
+
+export default {
+  name: 'copyToClipboard',
+  data () {
+    return {
+      ephemeral: {
+        tooltipActive: false
+      }
+    }
+  },
+  components: {
+    Tooltip
+  },
+  props: {
+    textToCopy: {
+      type: String,
+      required: true
+    },
+    tooltipText: {
+      type: String,
+      default: 'Copied to clipboard!'
```

Comment:
Missing L function

Reply:
In the changed code, I'm going to pass this text to `i18n.vue`. so I guess using `i18n` with `{{ }}` (like you pointed out above) would be enough? like this. :)
`i18n {{ tooltipText }}`

Reply:
i18n is only for translating string literals. Since `tooltipText` is not a string literal, but should not be passed to i18n or L.

Reply:
Thanks for the reminder! :)

Reply:
I just pushed commit where I used a computed property for translation. would you have a look at it?